cdserver/features/diagnostics/system-metadata.feature

```diff
@@ -7,5 +7,5 @@ Feature: System Metadata
     When I navigate to the url /info
     Then I should receive the response
       | name     | capdash2  |
-      | version  | x.y.z     |
-      | hostname | localhost |
```

Comment:
please include the rails version and other info that you added.  Thats all good stuff, let's list out the entire payload